in_windows_exporter_metrics: Add document for process metrics (#1182)
fluent/fluent-bit-docs: pipeline/inputs/windows-exporter-metrics.md

| we.service.where           | Specify the WHERE clause for retrieving service metrics.  | `NULL`   |
| we.service.include         | Specify the key value pairs for the include condition for the WHERE clause of service metrics. | `NULL`   |
| we.service.exclude         | Specify the key value pairs for the exclude condition for the WHERE clause of service metrics. | `NULL`   |

> | we.process.allow\_process\_regex | Specify the regex for process metrics to allow collection of. Collect all by default. | "/.+/"    |

Reviewer: Probably just reword slightly:
```suggestion
| we.process.allow\_process\_regex | Specify the regex covering the process metrics to  collect. Collect all by default. | "/.+/"    |
```

Author: Ah, good idea. process is able to use verb so your suggestion makes clear its meaning.
